### ISSUE TITLE
Nails and his leather jacket should be inseparable

### ISSUE THREAD

**Issue by Moddi** [ja2-stracciatella/ja2-stracciatella#1987]
### What is wrong

Nails don't want to be separated from his vest. Yet, when you hire him without equipment, he starts with no armor. When he then picks one up, it is treated as his leather jacket, which means you can't take it off.

A solution could be to make it so, that even when hired without equipment, he starts with a leather jacket. The same applies to when he is fired, or his contract expires, he should take his current leather jacket with him, so when he is rehired he still has it (or he gains a new one when he is back at A.I.M.).

Another possibility is to treat his jacket with C18 and Royal Jelly, which would result in destroying the vest and he can wear any other armor vest. This would probably result in the same bug (not tested), that this new armor would be treated as his unremovable jacket.

A solution could be, that a dialog window pops up, saying something like "This would destroy Nails leather jacket. Nails will not allow this." and aborts that action.

| JA2S version | Game version        | Vanilla bug? | Used to work? |
| ------------ | ------------------- | ------------ | ------------- |
|  0.19 | 1.12 GER  |    yes   |    no     |

**Comment by bezdonas** [comment]
If I remember correctly, he is still approving a change to Leather vest from Angel in San Mona and that change is not a bug.

**Comment by Moddi** [comment]
> If I remember correctly, he is still approving a change to Leather vest from Angel in San Mona and that change is not a bug.

Yes, but that I wasn't referring to. I specifically meant when you removed either of the vests via bug or overlook (dissolving the vest with C18 and royal jelly is only possible with the kevlar leather jacket anyway), you can put any armor on him, which was not intended by the game.
